alembic/versions/4f2f825bcf4a_disallow_null_values_in_autokarma.py

```diff
@@ -0,0 +1,34 @@
+"""Disallow NULL values in autokarma.
+
+Revision ID: 4f2f825bcf4a
+Revises: 3cde3882442a
+Create Date: 2016-10-19 15:09:50.884185
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '4f2f825bcf4a'
+down_revision = '3cde3882442a'
```

Comment:
`down_revision` should now be modified to `'06aa0e8aa5d2'` since another migration is recently added for dropping `update.karma` column https://github.com/fedora-infra/bodhi/commit/2a6bb6fc31a48914a46fe35915caad8829404905